config/nrfconnect/chip-module/Kconfig

```diff
@@ -22,6 +22,10 @@ config CHIP
 
 if CHIP
 
+choice LIBC_IMPLEMENTATION
```

Comment:
You may have to put this in `Kconfig.defaults` to make sure this default overrides the Zephyr's default.